_config.yml

```diff
@@ -30,6 +31,8 @@ baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://guide.memo.bank" # the base hostname & protocol for your site, e.g. http://example.com
 github_username: memobank
 email: contact@memo.bank
+markdown: GFM
+highlighter: rouge
```

Comment:
Since we removed rouge from the project and won't need this, we should remove this line.

Reply:
No biggie after all.

Reply:
Okay. Will do.